### DISCUSSION TITLE
What are points, etc?

### DISCUSSION THREAD

**Discussion by Rc85** [animir/node-rate-limiter-flexible#120] [Q&A] [answered]
What are points and consumption of points?

**Comment by animir** [comment, marked as answer]
Points are the main resource in the universe of limiters. If you have points here, you are rich, respected and can do anything you want. Every action you do consumes one or more points. And when all your points consumed, you are not allowed to do anything anymore. Fortunately, once per `duration` seconds in this beautiful and fare universe, you get all your points back.

There are more rules, though. You never know, until start living here.
